app/src/features/rules/screens/rulesList/components/RulesList/components/CharlesImporter/index.tsx

```diff
@@ -74,30 +76,36 @@ export const ImportFromCharlesModal: React.FC<ModalProps> = ({ isOpen, toggle, t
     trackCharlesSettingsImportViewed(triggeredBy);
   }, [triggeredBy]);
   return (
-    <RQModal open={isOpen} centered onCancel={toggle} className="import-from-charles-modal">
-      <div className="rq-modal-content">
-        <ImportFromCharles modalSrc={triggeredBy} callBack={() => toggle()} />
-      </div>
-    </RQModal>
+    <Modal open={isOpen} centered onCancel={toggle} footer={null} className="import-from-charles-modal custom-rq-modal">
+      <ImportFromCharles modalSrc={triggeredBy} callBack={() => toggle()} />
+    </Modal>
   );
 };
 
-interface ImportFromCharlesProps {
-  modalSrc?: string | null; // null indicates this is not mounted inside modal
-  callBack?: () => void;
-}
-
 export const ImportFromCharlesWrapperView: React.FC = () => {
   useEffect(() => {
     trackCharlesSettingsImportViewed("TOP_LEVEL_ROUTE");
   }, []);
   return (
-    <div className="root-wrapper">
+    <div className="charles-import-wrapper">
       <ImportFromCharles />
     </div>
   );
 };
-const ImportFromCharles: React.FC<ImportFromCharlesProps> = ({ modalSrc = null, callBack }) => {
+
+interface ImportFromCharlesProps {
+  modalSrc?: string | null; // null indicates this is not mounted inside modal
```

Comment:
We should give proper full names to the `props`. Lets rename this to either `modalSource` or `source`